### PR TITLE
Add filtering to inspect policy command

### PR DESCRIPTION
**Pull request by lcarva**
https://issues.redhat.com/browse/HACBS-1733

### CODE REVIEW

File: cmd/inspect/inspect_policy.go [hunk 1]
```diff
@@ -119,10 +128,65 @@ func inspectPolicyCmd() *cobra.Command {
...
 
 	if err := cmd.MarkFlagRequired("source"); err != nil {
 		panic(err)
 	}
 
 	return cmd
 }
+
+func filterResults(results map[string][]*ast.AnnotationsRef, rule, pkg, collection string) (map[string][]*ast.AnnotationsRef, error) {
+	if rule == "" && collection == "" {
```
Comment (zregvart) [inline cmd/inspect/inspect_policy.go:143]: I guess `pkg` as well?

```suggestion
	if rule == "" && pkg == "" && collection == "" {
```

**Review by zregvart — APPROVED** [review]
LGTM, found one logic issue

File: cmd/inspect/inspect_policy.go (hunk 1 above)
Comment (lcarva) [reply cmd/inspect/inspect_policy.go]: Thank you!

**Review by simonbaird — APPROVED** [review]

**Comment by lcarva** [comment]
I've been noticing the random 404 errors in the acceptance tests happening much more often now. :confused:

**Pull request merged**